_studio/mfx_lib/encode_hw/mpeg2/src/mfx_mpeg2_encode_utils_hw.cpp

```diff
@@ -1880,8 +1891,16 @@ namespace MPEG2EncoderHW
             MFX_CHECK_UMC_STS (ret);
 
             mfxU32 bufferSizeInKB = ((brcParams.maxFrameSize + 999)/1000);
-            par->mfx.BRCParamMultiplier = (mfxU16)((bufferSizeInKB + 0x10000) / 0x10000);
```

Comment:
Not related to current commit:
What if multiplier was already set to different value?